include/treelite/tree.h

```diff
@@ -358,8 +358,7 @@ class Tree {
   inline const char* GetFormatStringForNode();
   inline void GetPyBuffer(std::vector<PyBufferFrame>* dest);
   inline void SerializeToFile(FILE* dest_fp);
-  inline void InitFromPyBuffer(std::vector<PyBufferFrame>::iterator begin,
-                               std::vector<PyBufferFrame>::iterator end);
+  inline void InitFromPyBuffer(std::vector<PyBufferFrame>::iterator& it);
```

Comment:
We probably want to avoid passing iterators by reference (see the Better Code discussion from June 15th for details).

Reply:
@wphicks Can you point me to the discussion? I am not aware of it.

Reply:
Will point you to it offline